### PR TITLE
Remaining features

### PR DESCRIPTION
**Pull request by jasmingacic**
@edevenport go ahead start reviewing while i'm adding hardware reservations.

### CODE REVIEW

File: lib/packet/models/device.rb [hunk 1]
```diff
@@ -100,8 +101,8 @@ def reboot
 
         def start
           requires :id
-          service.poweron_device(id)
-          true
+          response = service.poweron_device(id)
+          true if response.status == 202
```
Comment (denza) [inline lib/packet/models/device.rb]: Suggesting to use response.status == 202 as return statment

**Comment by denza** [comment]
Table of Content is missing in the documentation

File: README.md [hunk 2]
```diff
@@ -731,9 +820,306 @@ Provides a collection of licenses for a given project.
...
+```ruby
+hardware_reservation.move(project_id) 
+```
+
+## Unit tests
+
+Run mock tests:
+
+```bash
+FOG_MOCK=true rake test
```
Comment (denza) [inline README.md]: It is possible to the different environments I had to use env FOG_MOCK=true rake test

File: lib/packet/models/device.rb (hunk 1 above)
Comment (jasmingacic) [reply lib/packet/models/device.rb]: If I use that rubocop compalins with 
```
Operator `==` used in void context. (warning:Lint/Void)
```

File: README.md [hunk 3]
```diff
@@ -731,9 +820,306 @@ Provides a collection of licenses for a given project.
...
+Creates a payment method.
+
+```ruby
+options = {
+    :name => "test",
+    :default => "false",
+    :nonce => "12"
+}
+
+response = compute.create_payment_method(org_id, options)
```
Comment (edevenport) [inline README.md]: This example should be use the model like the other example rather than the request.

File: README.md (hunk 2 above)
Comment (edevenport) [reply README.md]: I have ever had to begin a command with `env` to set environment variables. I think the existing command should be sufficient for most users.

File: tests/packet/models/test_capacity.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+# TestCapacity
+class TestCapacity < Minitest::Test
+  def self.test_order
+    :alpha
+  end
+
+  def setup
+    # Establish Connection
+    @compute = Fog::Compute::Packet.new(:packet_token => ENV["PACKET_TOKEN"])
```
Comment (edevenport) [inline tests/packet/models/test_capacity.rb]: Can we inherit from PACKET_TOKEN from the environment by default and allow the keyword `packet_token` as an optional override? If I already have the env variable defined, I'd rather not have to also specify the keyword.

**Pull request merged**